pom.xml

```diff
@@ -247,6 +247,15 @@ under the License.
       <version>${resolverVersion}</version>
       <scope>provided</scope>
     </dependency>
+    <!-- incremental build support (http://www.eclipse.org/m2e/documentation/m2e-making-maven-plugins-compat.html) -->
```

Comment:
Looks like - not up today link ...

Reply:
yes, reported at https://github.com/eclipse-m2e/m2e-core/issues/831#issuecomment-1294846414

Reply:
Btw. the website is back and the link works again.